plugins/browser-plugin-privacy-sandbox/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { BrowserPlugin } from '@snowplow/browser-tracker-core';
+import { PRIVACY_SANDBOX_TOPICS_SCHEMA } from './schemata';
+import { LOG } from '@snowplow/tracker-core';
+import { BrowsingTopicsOptions, PrivacySandboxTopic, Topic } from './types';
+
+declare global {
+  interface Document {
+    /* Declare Feature Policy types https://developer.mozilla.org/en-US/docs/Web/API/FeaturePolicy */
+    featurePolicy?: {
+      allowsFeature: (feature: string, origin?: string) => boolean;
+    };
+    browsingTopics: (options: BrowsingTopicsOptions) => Promise<PrivacySandboxTopic[]>;
+  }
+}
+
+let topics: Topic[] = [];
+
+/**
+ * Adds Privacy Sandbox Topics context to events
+ */
+export function PrivacySandboxPlugin(): BrowserPlugin {
+  return {
+    activateBrowserPlugin: function () {
+      const isTopicsApiAvailable =
+        'browsingTopics' in document && document.featurePolicy?.allowsFeature('browsing-topics');
+
+      if (isTopicsApiAvailable) {
+        document
+          .browsingTopics({ skipObservation: true })
+          .then((availableTopics) => {
+            topics = availableTopics.map(({ topic, version }) => ({
+              topic,
```

Comment:
Do you mean, not the full information ?
The `version` attribute includes the full set of information as far as I can understand which can then be split at the level we want in the model.

https://patcg-individual-drafts.github.io/topics/#terminology-and-types-header
> Given [configuration version](https://patcg-individual-drafts.github.io/topics/#browsing-topics-types-configuration-version) configurationVersion, [taxonomy version](https://patcg-individual-drafts.github.io/topics/#browsing-topics-types-taxonomy-version) taxonomyVersion, and [model version](https://patcg-individual-drafts.github.io/topics/#browsing-topics-types-model-version) modelVersion, the version is the result of [concatenating](https://infra.spec.whatwg.org/#string-concatenate) « configurationVersion, taxonomyVersion, modelVersion » using ":".

Reply:
Oh interesting, I was looking at https://developer.chrome.com/docs/privacy-sandbox/topics/#epoch which says it's just `configurationVersion` and `modelVersion` (so no `taxonomyVersion`). If it's all three then all good. Have you tested this locally to confirm the current implementation matches the draft spec instead of the other one?

Reply:
Based on the OT yes, testing on Chrome 115 as well.